docker-compose.core.yml

```
@@ -42,7 +42,7 @@ services:
   ui: # TODO: change the name since to app or lb since it's the main entry point for every request
     build:
       context: ui
-      dockerfile: docker/Dockerfile-ui
+      dockerfile: docker/Dockerfile
     ports:
      - "8081:8081"
```

You can remap to 8080 here
```suggestion
      - "8081:8080"
    volumes:
      - ${PWD}/ui/docker/config/local.conf:/etc/nginx/conf.d/default.conf
```